lib/tunnel.js

```diff
@@ -96,13 +97,25 @@ function Tunnel() {
               self.binaryPath = binaryPath;
               // Run binary after getting this and attempting atmost 5 times
               console.log(`Starting tunnel`);
-              runBinary_(self, 5, function(e, response) {
-                if (e) {
-                  return reject(e);
-                } else {
-                  return resolve(response);
+              if (options.detachedMode === 'true') {
+                console.log("Running Tunnel in Detached Mode")
```

Comment:
Add the retry count in console log statement

Reply:
This log is there inside runBinaryV2_ function